src/Elliptic/Systems/Elasticity/Tags.hpp

```diff
@@ -0,0 +1,60 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+/*!
+ * \ingroup EllipticSystemsGroup
+ * \brief Items related to solving linear elasticity problems
+ *
+ * \details In linear elasticity problems we solve for the displacement vector
+ * field \f$\boldsymbol{u}\f$ in an elastic material that responds to an applied
+ * force \f$\boldsymbol{F}\f$. In this static approximation the equation of
+ * motion reduces to the elliptic equation \f[
+ * -\Nabla_i T^{ij} = F^j
+ * \f]
+ * where the stress \f$T^{ij}\f$ is related to the strain
+ * \f$S_{ij}=\Nabla_{(i}u_{j)}\f$ by a linear constitutive relation
+ * \f$T^{ij}=\chi^{ijkl}S_{kl}\f$ that describes the elastic properties of the
+ * material.
+ */
+namespace Elasticity {
+
+/*!
+ * \brief The amount of material deformation \f$\boldsymbol{u}(x)\f$ when a
+ * static force is applied.
+ */
+template <size_t Dim>
```

Comment:
I can't say if that will become necessary any time soon. Therefore I'd prefer not to add that complication.

Reply:
Sounds good.

Reply:
Since these are elastostatic problems, I think an inertial frame should suffice wherever you need to specify a frame. Also, you'll always solve the equations on an extended domain, so I don't see the point of templating on DataType to support doubles.